source/material_model/rheology/peierls_creep.cc

```diff
@@ -559,14 +542,12 @@ namespace aspect
                                                                   "Peierls glide parameters q",
                                                                   true,
                                                                   expected_n_phases_per_composition);
-        stress_c = Utilities::parse_map_to_double_array(prm.get("Cutoff stress"),
+        stress_coff = Utilities::parse_map_to_double_array(prm.get("Cutoff stress for Peierls creep"),
```

Comment:
Please add a comment above this line to explain in more detail what this parameter is doing. Something similar to the changelog would be good: 

`// At stresses below stress_cutoff, the strain rate is modelled as a quadratic function of the stress (edot_ii = astress^2 + bstress).`
`// This parameter is sometimes necessary to prevent negative viscosities at low strain rates when the Peierls' creep flow law has a power law stress exponent equal to zero.`